geth/api/api.go

```diff
@@ -49,9 +50,9 @@ func (api *StatusAPI) JailManager() jail.Manager {
 	return api.b.JailManager()
 }
 
-// TxQueueManager returns reference to account manager
-func (api *StatusAPI) TxQueueManager() *transactions.Manager {
-	return api.b.TxQueueManager()
+// TxFactory returns reference to account manager
```

Comment:
not yours, but the comment says `returns reference to account manager` should say transaction factory instead?